packages/eslint-config-kibana/.eslintrc.js

```diff
@@ -118,8 +118,6 @@ module.exports = {
     'react/jsx-first-prop-new-line': ['error', 'multiline-multiprop'],
     'react/jsx-equals-spacing': ['error', 'never'],
     'react/jsx-indent': ['error', 2],
-    'react/no-did-mount-set-state': 'error',
-    'react/no-did-update-set-state': 'error',
```

Comment:
Not related to the React bump but thought I'd put it here:
Are we still trying to follow airbnb's convention (with minor modifications) or do we see our style guide as very different?

If we still intent to follow airbnb style, wouldn't it be nicer if we extended their rules? Their react rules look pretty similar: 
https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/rules/react.js